apps/wallet-mobile/src/features/Discover/DiscoverNavigator.tsx

```diff
@@ -42,6 +43,12 @@ export const DiscoverNavigator = () => {
         </Stack.Screen>
 
         <Stack.Screen name="discover-browser" component={BrowserNavigator} options={{headerShown: false}} />
+
+        <Stack.Screen
+          name="discover-review-tx"
+          component={ReviewTransaction}
+          options={{...defaultStackNavigationOptions(atoms, color), title: strings.transactionReview}}
```

Comment:
is this route not catching `apps/wallet-defaultStackNavigationOptions` from navigator? 🤔